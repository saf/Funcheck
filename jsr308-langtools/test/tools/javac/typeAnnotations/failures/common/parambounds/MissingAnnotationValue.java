/*
 * @test /nodynamiccopyright/
 * @bug 6843077
 * @summary check for missing annotation value
 * @author Mahmood Ali
 * @compile/fail/ref=MissingAnnotationValue.out -XDrawDiagnostics -source 1.7 MissingAnnotationValue.java
 */
class MissingAnnotationValue<K extends @A Object> {
}

@interface A { int field(); }
