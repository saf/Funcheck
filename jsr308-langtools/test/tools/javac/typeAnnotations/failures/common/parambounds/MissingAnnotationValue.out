MissingAnnotationValue.java:8:40: compiler.err.annotation.missing.default.value: A, field
1 error
